extensions/wikia/PortableInfobox/services/Helpers/SimpleXmlUtil.php

```diff
@@ -28,8 +28,12 @@ public static function getInstance() {
 	public function getInnerXML( $node ) {
 		$innerXML = '';
 		if ( $node instanceof \SimpleXMLElement ) {
-			foreach ( dom_import_simplexml( $node )->childNodes as $child ) {
-				$innerXML .= $child->ownerDocument->saveXML( $child );
+			$domDocument = dom_import_simplexml( $node );
+
+			if ( $domDocument->childNodes->length ) {
```

Comment:
On second though, we'd need to have a two-sided conditional here
( if $domDocument instanceof DomNode && $domDocument->hasChildNodes() )

What do you think?